doc/user/content/integrations/monitor/datadog.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Monitor Materialize with Datadog"
+description: "Use Datadog to monitor Materialize."
+menu:
+  main:
+    parent: ""
+    name: 
+---
+
+Materialize exposes a system catalog that contains metadata about your running
+Materialize instances. The system catalog consists of four schemas available
+in all databases, which contain tables, and views that expose
+different types of metadata. 
+
+This metadata can help you track and monitor your Materialize usage in tools
+like Datadog.
+
+For this guide, you'll setup a Datadog integration and create a dashboard to
+monitor cluster performance, credit consumption, and storage usage.
+
+We'll also share some helpful queries you can use in your observability tools to
+keep track of other metrics important to you.
+
+## Setup
+
+Before you get started make sure you have the following:
+
+- Docker
+- Datadog API Key
+
+This integration quick start uses the Prometheus SQL exporter Docker image to
+pass data to and from your Materialize instance and Datadog agent.
+
+Clone the [example repository](https://github.com/MaterializeInc/demos/tree/main/integrations/datadog) and navigate to the `integrations/datadog`
+directory.
+
+Open the `config.yaml` file in your text editor and look for the `connections`
+settings.
+
+This `connections` setting monitors your Materialize cluster replica metrics for
+CPU, memory, credit usage, and replica status. There are also queries to monitor
+source and sink usage.
+
+```yaml
+...
+connections:
+- "postgres://<USER>:<PASSWORD>@<HOST>:<PORT>/materialize?sslmode=require"
+...
+```
+You can copy this string from the Connect page in your Materialize region.
+
+{{< note >}}
+You need to escape the special `@` character in your user email for a successful connection. For example, `name@email.com`, use `name%40email.com`.
+{{</ note >}}
+
+Create a new app password on the **Connect** page and paste it into the
+`<PASSWORD>` section of the connection command.
+
+Update the second `connections` setting near the end of the file. This
+connection queries your Materialize storage usage and reports on objects using
+storage resources.
+
+Edit the `docker-compose.yaml` file with your Datadog API key and region:
+
+```yaml
+...
+    environment:
+     - DD_API_KEY=<API_KEY> 
+     - DD_SITE=<DATADOG_REGION_ADDRESS>
+...
+```
+
+## Deploy the SQL exporter and Datadog agents
+
+In the current directory, run Docker to deploy the SQL exporter and the Datadog
+agents. The SQL exporter tool uses the connection string you created in a
+previous step to connect with your Materialize instance. 
+
+```shell
+$ docker compose up -d
+```
+
+The Prometheus SQL exporter container runs a service that executes queries against your
+database and then exports the metrics in a format that most monitoring services
```

Comment:
Only because a few lines after says `the Datadog agent with the OpenMetrics...`.
```suggestion
database and then exports the metrics using the OpenMetrics standard.
```